### PR TITLE
refactor: resolve vscode type errors in wakatime card render and remove redundant css

### PR DESCRIPTION
**Pull request by qwerty541**

### CODE REVIEW

**Comment by rickstaa** [comment]
@qwerty541, thanks for the pull request. Here, I would do one of the following:

1. Remove the global `getStyles` function and replace it with a `getRankCardStyle` and `getWakatimeCardStyle` function inside the wakatime and stats card files.
2. OR: modify the global `getStyles` function to have a `card_type` variable.

Your current PR causes two `getStyles` functions to exist in the codebase, which is not bad but might confuse people. I prefer option 1, but since @anuraghazra initially added this function, he may give his thoughts before we merge this 👍🏻.

**Review by qwerty541 — COMMENTED** [review]
> @qwerty541, thanks for the pull request. Here, I would do one of the following:
> 
> 1. Remove the global `getStyles` function and replace it with a `getRankCardStyle` and `getWakatimeCardStyle` function inside the wakatime and stats card files.
> 2. OR: modify the global `getStyles` function to have a `card_type` variable.
> 
> Your current PR causes two `getStyles` functions to exist in the codebase, which is not bad but might confuse people. I prefer option 1, but since @anuraghazra initially added this function, he may give his thoughts before we merge this 👍🏻.

@rickstaa Thanks for review, i agree with your thoughts about removing global `getStyles` function. I have updated the code, check it out.

File: src/cards/stats-card.js [hunk 1]
```diff
@@ -76,6 +75,118 @@ const createTextNode = ({
...
+      }
+      to {
+        stroke-dashoffset: ${calculateCircleProgress(progress)};
+      }
+    }
+  `;
+};
+
+/**
+ * Retrieves CSS styles for a card.
```
Comment (rickstaa) [inline src/cards/stats-card.js:119]:
```suggestion
 * Retrieves CSS styles for the card.
```

File: src/cards/wakatime-card.js [hunk 2]
```diff
@@ -158,6 +157,36 @@ const recalculatePercentages = (languages) => {
   });
 };
 
+/**
+ * Retrieves CSS styles for a card.
```
Comment (rickstaa) [inline src/cards/wakatime-card.js:161]:
```suggestion
 * Retrieves CSS styles for the card.
```

**Review by rickstaa — APPROVED** [review]

**Pull request merged**